Documentation/Settings.md

```diff
@@ -0,0 +1,302 @@
+<!-- This is an automatically generated file. DO NOT EDIT. Edit gen_menu_docs.py instead -->
+
+# IronOS Settings Menu
+
+The below breaks down the menu's and what each setting means.
+    
+## Menu Categories
+
+In the menu there are a few main categories that are used to keep the list manageable.
+
+### Category: Power settings
+
+Menu for settings related to power. Main settings to do with the input voltage.
+
+### Category: Soldering settings
+
+Setttings for soldering mode, such as tip temperature, boost temps, sleep temps.
+
+### Category: Sleep mode
+
+Settings to do with power saving, such as sleep mode and shutdown modes.
+
+### Category: User interface
+
+User interface related settings, such as Units.
+
+### Category: Advanced settings
+
+Advanced settings. Misc catchall for settings that dont fit anywhere else or settings that require some thought before use.
+
+## Settings
+
+These are all of the settings possible in the menu.
+**Not all settings are visible for all devices.**
+For example, the TS100 does not have USB-PD settings.
+
+When using the device, if unsure you can pause (press nothing) on a setting and after a short delay help text will scroll across the screen.
+This is the "on device help text".
+
+### Setting: Power source
+
+When device is powered by a battery, adjusts the low voltage threshold for when the unit turns off the heater to protect the battery.
+
+On device help text:
+
+Power source. Sets cutoff voltage. (DC 10V) (S 3.3V per cell, disable power limit)
+
+### Setting: Sleep temp
+
+Temperature the device will drop down to while asleep. Typically around halfway between off and soldering temperature.
+
+On device help text:
+
+Tip temperature while in "sleep mode"
+
+### Setting: Sleep timeout
+
+How long after inactivity the device drops down to the sleep temperature.
```

Comment:
`Period of inactivity after which` the device drops down to the sleep temperature.